_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+
+#### Immediate Actions
+
+##### Do Not Invest in Re-Work
+
+Presently there are many SaaS offerings that provide many useful tools that would assist ESDC in increasing its efficiency in order to better deliver its services to Canadians. By developing solutions that already exist, the government of Canada is investing in unnecessary rework in order to produce substandard products, due to being unable to benefit from the economies of scale SaaS offerings are able to leverage. For example, it wouldn't be responsible stewardship of public funds to invest long-time full-time development resources to a survey tool. However, existing SaaS offerings are able to commit full-time resources over a long period of time to this relatively small function, and continuously improve their service offering. As a result, they are able to allocate significant amounts of time into perfecting a simple and reusable function, such as the creation of a survey. ESDC should permit itself to benefit from such offerings and instead focus on developing solutions that directly deliver on its mandate.
+
+This claim is acknowledging the many exception cases there SaaS offerings may not fully meet the requirements set by the Government of Canada, for example the [Accessible Canada Act](https://www.parl.ca/DocumentViewer/en/42-1/bill/C-81/royal-assent) or [Direction for Electronic Data Residency](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/direction-electronic-data-residency.html).
+
+It is for this reason that, in the [Future Actions](#FutureActions) section ESDC we discuss that ESDC should work relentlessly on improving its in-house development capabilities in order to eventually reduce the reliance on SaaS offerings.
+
+##### Leverage for Quick-Wins
+
+SaaS should be leveraged for 'quick wins' where only unprotected data is being used and for where it is not intended to contact citizens directly. Further, it should be leveraged when investment in an in-house developed solution would be in misalignment with responsible stewardship of public funds given the benefits said in-house delivery would deliver.
+
+> [Federal public servants are entrusted to use and care for public resources responsibly, for both the short term and long term.](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049)
+
+#### Future Actions
+
+##### Reduce Reliance on SaaS
+
+[TO-DO: DevOps Handbook Reference]
+
+SaaS can save time and money by not investing resources into the creation of existing tools.
+
+In-house development, due to heavy governance and approval processes, is a lengthy and thusly expensive process. However, as an organization
+
+## Conclusion
+
+### Answering the question: SaaS or Develop Cloud Based Solution?
```

Comment:
That's a GREAT section
Some of that wording should be in the context section. Sorted dissert that initial flawed question with what the intent behind those flawed words were: SaaS or develop Cloud Based solutions